### PR TITLE
Update devenvs

### PR DESCRIPTION
**Pull request by kse3hi**
# Description

* Integrate devenv-devcontainer-setup v1.1.6
* Integrate devenv-github-workflows v1.0.16

## Checklist

* [ ] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [ ] Vehicle App can process MQTT messages and call the seat service
* [ ] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [x] Devcontainer can be opened successfully
* [x] Devcontainer can be opened successfully behind a corporate proxy
* [x] Devcontainer can be re-built successfully

* [x] Release workflow is passing

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ghcr.io/eclipse-velocitas/devcontainer-base-images/cpp:latest
+FROM ghcr.io/eclipse-velocitas/devcontainer-base-images/cpp:v0.0.1
```
Comment (BjoernAtBosch) [inline .devcontainer/Dockerfile]: Why having this small version number for the base image?
I'd suggest using at least v0.1.0 or even v1.0.0 ...

**Review by BjoernAtBosch — COMMENTED** [review]
Just "complaining" about version number of base image.
The remainder LGTM

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (kse3hi) [reply .devcontainer/Dockerfile]: Good question. We have just never released it before, and just decided to have the very first version

**Review by doosuu — APPROVED** [review]
LGTM

**Pull request merged**